test/web/10-CredentialCard.js

```diff
@@ -11,19 +11,55 @@ function renderCredentialCard(propsData) {
 }
 
 describe('CredentialCard', () => {
-  it('should check something on the rendered element', async () => {
-    // TODO: specify propsData
+  it('should render content if type of value is not falsy in credential' +
+    'subject', async () => {
     const vm = renderCredentialCard({
       credential: {
-        name: 'Test Credential',
+        credentialSubject: {
+          birthCountry: 'Mexico',
+        },
         type: ['TestCredential'],
         description: 'Test description',
         image: 'http://example.com/some-image.png'
       },
-      schema: {}
+      schema: {
+        birthCountry: {
+          name: 'Birth Country'
+        }
+      }
     });
     should.exist(vm);
     should.exist(vm.$el);
-    // TODO: $vm.el.something
+    vm.$el.querySelector(
+      '.q-item__section.g-field-data-regular.q-item__section--main')
```

Comment:
This query is dangerous because it depends on Quasar's internal API. These internals can be changed at any time. We need to either pin to a specific version of Quasar for these tests or even better... find another way that doesn't depend on these classes.

Reply:
hmm, I updated to use only the class `.g-field-data-regular` since it's a class that we set to the value field.

Reply:
@JSAssassin Great. Thank you for working on this.